activate.sh

```diff
@@ -36,3 +36,11 @@ You can manage your cluster by opening the web application running on
 port 80 of this node from your browser.
 
 EOF
+
+# Generate root ssh key and share it with velum
+# https://bugzilla.suse.com/show_bug.cgi?id=1030876
+if ! [ -f /root/.ssh/id_rsa ]; then
+  ssh-keygen -b 2048 -f /root/.ssh/id_rsa -t rsa -N ""
+fi
+[ -d /tmp/ssh ] || mkdir /tmp/ssh
+cp /root/.ssh/id_rsa.pub /tmp/ssh
```

Comment:
This was discussed on caasp mailing list, and I tried also submit this change ttps://build.suse.de/request/show/132262. Thorsten said that this should be done in`activate.sh` script